CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ to [Semantic Versioning](http://semver.org/).
     accessible.
 
 ### Changed
+- The homepage has been refreshed to better inform
+the site vistors what they can do on the site.
```

Comment:
Aw yeah, right up at the top! :yum: 

Er can you make the second line indented so it is flush with the first letter of the previous line? Like all the other bullet points below are? Also mention that it shows the latest BLOG POSTS!!!!!!!!!!!